tests/integration/conftest.py

```diff
@@ -993,8 +1317,7 @@ def search_qtest(qtest_api_token, qtest_project_id, object_type, query, fields=N
     except requests.exceptions.RequestException as e:
         raise RuntimeError("The qTest API reported an error!\n"
                            "Status code: {}\n"
-                           "Reason: {}\n"
-                           "Message: {}".format(e.status, e.reason, e.body))
+                           "Reason: {}\n".format(e.response.status_code, e.response.reason))
```

Comment:
Discovered a bug while testing. Used the wrong attributes on the caught exception object.